pyproject.toml

```diff
@@ -11,3 +11,9 @@ requires = [
     "numpy>=1.26.0b1; python_version>='3.12'"
 ]
 build-backend = 'setuptools.build_meta'
+
+[tool.cibuildwheel]
+
+[[tool.cibuildwheel.overrides]]
+select = "cp312-*"
+environment = "PIP_PRE=1"
```

Comment:
Does this mean all the wheels now use numpy 1.26rc or just cp312?

Reply:
Yes on 3.12 the wheels are tested against the numpy RC. Note that we don't build any Python 3.12 wheels so the build dependencies are not important there.

Reply:
🤪

Reply:
p.s. Are you going to do a similar patch for `astropy`?

Reply:
Haha unfortunately not, Cython's support for the limited API is very rudimentary